Netw4ppl/app/src/main/java/netw4ppl/ines/AddPersonActivity.java

```diff
@@ -75,26 +75,24 @@ protected void onCreate(Bundle savedInstanceState) {
         mRecyclerView.setAdapter(adapter);
         mRecyclerView.setLayoutManager(new LinearLayoutManager(this));
 
-        mButtonCancel.setOnClickListener(v -> {
-            this.finish();
-        });
+        mButtonCancel.setOnClickListener(v -> this.finish());
 
         mButtonSave.setOnClickListener(v -> {
 
             if (verificationInputPerson()) {
-
                 Calendar calendar = Calendar.getInstance();
                 /* Ajout d'une nouvelle personne */
                 if (new_person) {
-
                     // ajout du champ date indispensable pour la database
                     SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
                     String date_creation = dateFormat.format(calendar.getTime());
                     person.putInfo("date", date_creation);
+                    System.out.println(person.getInfoByKey("date"));
```

Comment:
TO be deleted before merging to preprod. Maybe you should add a tag to remember

Reply:
👍🏼